include/storage/FileserverFileLoader.h

```diff
@@ -30,6 +30,28 @@ class FileserverFileLoader : public FileLoader
 
     std::vector<uint8_t> loadSharedFile(const std::string& path) override;
 
+    std::vector<uint8_t> loadFunctionObjectHash(
```

Comment:
Using pure virtual functions like this is a bit nasty as you have to override them for every single subclass, but I didn't want to go down another rabbit hole refactoring the `FileLoader` class.